cluster/gce/manifests/kube-addon-manager.yaml

```diff
@@ -30,6 +30,9 @@ spec:
     - mountPath: /var/log
       name: varlog
       readOnly: false
+    env:
+    - name: KUBECTL_EXTRA_PRUNE_WHITELIST
+      value: {{kubectl_extra_prune_whitelist}}
```

Comment:
Just to confirm, an empty value is valid right?

Reply:
Yeah, it is valid.

But I'll change the code to make it more explicit.